File: Practice/src/introduction_coding_tests/PrimeFactorization.py
def solution(n):
    answer = []
    i = 2
    while i <= n:
        if n % i == 0:
            if len(answer) == 0 or answer[-1] != i:
                answer.append(i)
            n /= i
        else:
            i += 1
    return answer

File: Practice/src/introduction_coding_tests/PrimeFactorization.js
function solution(n) {
    var answer = [];
    for (let i = 2; i <= n; ) {
        if (n % i == 0) {
            if (!answer.includes(i)) {
                answer.push(i);
            }
            n /= i;
        } else {
            i++;
        }
    }
    return answer;
}
